agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/common/SystemInformation.java

```diff
@@ -32,18 +32,27 @@
 
   private static final String DEFAULT_PROCESS_NAME = "Java_Process";
 
+  private static final boolean WINDOWS;
+  private static final boolean LINUX;
+
+  static {
+    String os = System.getProperty("os.name").toLowerCase(Locale.ENGLISH);
+    WINDOWS = os.contains("win");
+    LINUX = os.contains("nux");
```

Comment:
I updated the logic to match the apache commons logic for `IS_OS_LINUX` and `IS_OS_WINDOWS`. Since we were only using the data to populate "linux" previously anyways, I think this is good, and if we want to capture "mac" or some other flavor of unix separately we can add that later.